### PR TITLE
NOISSUE - Add README for scenarios

### PR DESCRIPTION
**Pull request by blokovi**

### CODE REVIEW

File: mzbench/README.md [hunk 1]
```diff
@@ -1 +1,58 @@
...
+List of scenarios that we will test.
+- [Fan-in](scenarios/fan_in.bdl)
+- [Fan-out](scenarios/fan_out.bdl)
+
+A prerequisite for running these test scenarios is to have Mainflux instance where you should:
+- create one Channel
+- create two Things
+- connect both Things to Channel
+
+Required paramaters are then set with running mzbench with --env param:
```
Comment (dborovcanin) [inline mzbench/README.md]: Typo: parameters.

File: mzbench/README.md [hunk 2]
```diff
@@ -1 +1,58 @@
...
+| --- | --- | --- |
+| MF_MZBENCH_MQTT_ENDPOINT | IP/domain of MQTT endpoint on Mainflux | 127.0.0.1 |
+| MF_MZBENCH_MQTT_PORT | Port of MQTT endpoint on Mainflux | 1883 |
+| MF_MZBENCH_CHANNEL_ID | ID of pre-provisioned channel on which both things are connected| |
+| MF_MZBENCH_THING_1_ID | First thing ID of pre-provisioned Mainflux thing | |
+| MF_MZBENCH_THING_1_KEY | First thing key of pre-provisioned Mainflux thing | |
+| MF_MZBENCH_THING_2_ID | Second thing ID of pre-provisioned Mainflux thing | |
+| MF_MZBENCH_THING_2_KEY | Second thing key of pre-provisioned Mainflux thing| |
+
+Scenarios have it's own parameters (like duration of test, rate of publishing...) that are set through environment variables, also. You can redefine an environment variable even during the benchmark without interrupting the benchmark itself with [change env](https://github.com/mzbench/mzbench/blob/master/doc/cli.md#change_env) command.
```
Comment (dborovcanin) [inline mzbench/README.md]: ...its own...

File: mzbench/README.md [hunk 3]
```diff
@@ -1 +1,58 @@
...
+
+## Fan-in
+
+In fan-in scenario numerous MQTT clients are sending messages on same Mainflux channel, but every client to exclusive subtopic on that channel. One subscriber is consuming all that messages.
```
Comment (dborovcanin) [inline mzbench/README.md]: ...all those messages.

File: mzbench/README.md [hunk 4]
```diff
@@ -1 +1,58 @@
...
+## Fan-out
+
+In fan-out scenario one MQTT client is sending messages on Mainflux channel. Numerous subscribers are connected to that channel and consuming  messages.
+
+| Environment variable | Description | Default |
+| --- | --- | --- |
+| MF_MZBENCH_SUB_NUM | Number of subscribers | 10000 |
+| MF_MZBENCH_PUB_RATE | Message rate in rps. Number of messages that publisher is sending per second | 1 |
+| MF_MZBENCH_PUB_TIME | Duration of publishing messages in minutes | 5 |
+| MF_MZBENCH_MSG_SIZE | Size of messages in bytes. Messages are sequence of random bytes | 100 |
```
Comment (dborovcanin) [inline mzbench/README.md]: ...are a sequence of...

**Review by dborovcanin — CHANGES_REQUESTED** [review]

File: mzbench/README.md (hunk 3 above)
Comment (nmarcetic) [reply mzbench/README.md]: Can you pls link fan-in and fan-out, wiki links.

**Review by nmarcetic — APPROVED** [review]

**Review by dborovcanin — APPROVED** [review]

**Review by mteodor — APPROVED** [review]
LGTM

File: mzbench/README.md [hunk 5]
```diff
@@ -1 +1,58 @@
...
+
+| Environment variable | Description | Default |
+| --- | --- | --- |
+| MF_MZBENCH_MQTT_ENDPOINT | IP/domain of MQTT endpoint on Mainflux | 127.0.0.1 |
+| MF_MZBENCH_MQTT_PORT | Port of MQTT endpoint on Mainflux | 1883 |
+| MF_MZBENCH_CHANNEL_ID | ID of pre-provisioned channel on which both things are connected| |
+| MF_MZBENCH_THING_1_ID | First thing ID of pre-provisioned Mainflux thing | |
+| MF_MZBENCH_THING_1_KEY | First thing key of pre-provisioned Mainflux thing | |
+| MF_MZBENCH_THING_2_ID | Second thing ID of pre-provisioned Mainflux thing | |
+| MF_MZBENCH_THING_2_KEY | Second thing key of pre-provisioned Mainflux thing| |
```
Comment (manuio) [inline mzbench/README.md:30]: Can you add some tabulation here and for other tables please?

File: mzbench/README.md [hunk 6]
```diff
@@ -1 +1,58 @@
...
+
+Folder [scenarios](scenarios/) contains .bdl files ([Benchmark Definition Language](https://github.com/mzbench/mzbench/blob/master/doc/scenarios/spec.md)) that represents different testing scenarios for MZbench.
+
+
+List of scenarios that we will test.
+- [Fan-in](scenarios/fan_in.bdl)
+- [Fan-out](scenarios/fan_out.bdl)
+
+A prerequisite for running these test scenarios is to have Mainflux instance where you should:
+- create one Channel
```
Comment (drasko) [inline mzbench/README.md]: Capitalize the bullets

**Review by drasko — CHANGES_REQUESTED** [review]

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**